src/test/java/com/google/api/codegen/testdata/ruby/ruby_main_library.baseline

```diff
@@ -222,12 +218,13 @@ module Library
         if app_name || app_version
           warn "`app_name` and `app_version` are no longer being used in the request headers."
         end
+        if service_path || port
+          warn "`service_path` and `port` parameters will be removed"
```

Comment:
nit: can you change the wording here and above to "... are deprecated and will be removed"?